lib/pluginmanager/prepare_offline_pack.rb

```diff
@@ -0,0 +1,60 @@
+# encoding: utf-8
+require "pluginmanager/command"
+require "pluginmanager/errors"
+
+class LogStash::PluginManager::PrepareOfflinePack < LogStash::PluginManager::Command
+  parameter "[PLUGIN] ...", "plugin name(s)", :attribute_name => :plugins_arg
+  option "--output", "OUTPUT", "output file", :default => ::File.join(LogStash::Environment::LOGSTASH_HOME, "logstash-offline-plugins-#{LOGSTASH_VERSION}.zip")
+
+  def execute
+    validate_arguments!
+
+    # We need to start bundler, dependencies so  the plugins are available for the prepare
+    LogStash::Bundler.setup!({:without => [:build, :development]})
+
+    # manually require paquet since its an external dependency
+    require "pluginmanager/offline_plugin_packager"
+    require "paquet"
+    require "paquet/shell_ui"
+
+    # Override the shell output with the one from the plugin manager
+    # To silence some of debugs/info statements
+    Paquet.ui = Paquet::SilentUI unless debug?
+
+    FileUtils.rm_rf(output) if ::File.exist?(output)
+
+    LogStash::PluginManager::OfflinePluginPackager.package(plugins_arg, output)
+
+    message = <<-EOS
+Offline package created at: #{output}
+
+You can install it with this command `bin/logstash-plugin install file:///PATH_TO/#{::File.basename(output)}`
```

Comment:
Can we show the entire output path? 

You can install it with this command `bin/logstash-plugin install file:///{::File.path(output)}`